main.py

```diff
@@ -281,6 +283,12 @@ def create_circuit(self, request: Request) -> JSONResponse:
         except ValueError as exception:
             raise HTTPException(400, detail=str(exception)) from exception
 
+        try:
+            self._check_no_tag_duplication(evc.id, evc.uni_a, evc.uni_z)
+        except DuplicatedNoTagUNI as exception:
+            log.debug("update result %s %s", exception, 409)
```

Comment:
Trivial detail but this log entry with "update result" might get confusing, looks like it came from the update part, if you want to maintain it here too maybe "create_circuit" like the other parts are using? @Alopalao

Reply:
Fixed log.